src/istio/charts/uds-istio-egress-config/templates/configmap.yaml

```
@@ -0,0 +1,61 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: egress-waypoint-config
+  namespace: {{ .Release.Namespace }}
+data:
+  {{- with .Values.config.deployment }}
+  deployment: |
+    {{- with .podAnnotations }}
+    metadata:
+      annotations: 
```

nit: cleanup whitespace
```suggestion
      annotations:
```